src/atoms/Button/button.module.css

```diff
@@ -0,0 +1,21 @@
+.btn {
+  border-style: solid;
+  border-width: 3px;
+  border-radius: 8px;
+  padding: 8px 10px;
```

Comment:
We need to use `calc()` and `var(--layout-base)` here.

Reply:
Done.